src/osal/osal_keys_unix.c

```diff
@@ -1,22 +1,222 @@
 #include "osal_keys.h"
 
+#ifdef OS_LINUX
```

Comment:
All these #ifdef OS_LINUX look ugly. If this implementation is Linux-specific, it is better to split the file on two parts
#ifdef OS_LINUX
Linux implementation
#else
Dummy implementation
#endif